src/gcam/components.py

```diff
@@ -1,39 +1,37 @@
-"""Definitions for the modules for GCAM and associated downstream models.
+"""Definitions for the components for GCAM and associated downstream models.
 
-We use the word "module" here in a bit of a nonstandard way.  A GCAM
-"module" is a functional unit that performs a specific step in the
-GCAM processing pipeline.  These modules are distinct from python
-modules; a GCAM module may be implemented in one or several python
-modules of its own (as in the water disaggregation module), or it may
-be completely contained in this module.
+A GCAM "component" is a functional unit that performs a specific step in
```

Comment:
I think we want to change the wording a bit here and elsewhere.  Although the framework is set up to support models from the GCAM ecosystem, it could be used for other things as well.

I'll update the wording when I'm done going through the rest of the diffs.